tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/deid/conflicting_hpo_state_generalization_test.py

```diff
@@ -0,0 +1,87 @@
+"""
+ Integration test for generalization of conflicting HPO States.
+
+ Original Issue: DC-512
+ """
+# Python imports
+import os
+
+# Project Imports
+from app_identity import PROJECT_ID
+from cdr_cleaner.cleaning_rules.deid.conflicting_hpo_state_generalization import ConflictingHpoStateGeneralize
+from common import JINJA_ENV, OBSERVATION, VOCABULARY_TABLES
+from tests.integration_tests.data_steward.cdr_cleaner.cleaning_rules.bigquery_tests_base import BaseTest
+
+INSERT_RAW_DATA = JINJA_ENV.from_string("""
+   INSERT INTO `{{project_id}}.{{dataset_id}}.observation` (
+       person_id,
+       src_hpo_id
+       )
+     VALUES
+       (1,'hpo_100'),
+       (2,'rdr'),
+       (3,'rdr'),
+       (4,'hpo_103')
+ """)
+
+
+class ConflictingHpoStateGeneralizeTest(BaseTest.CleaningRulesTestBase):
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        print('**************************************************************')
+        print(cls.__name__)
+        print('**************************************************************')
+
+        super().initialize_class_vars()
+
+        # Set the test project identifier
+        cls.project_id = os.environ.get(PROJECT_ID)
+
+        # Set the expected test datasets
+        cls.dataset_id = os.environ.get('UNIONED_EHR_DATASET_ID')
```

Comment:
I think you want this to be
```suggestion
        cls.dataset_id = os.environ.get('UNIONED_DATASET_ID')
```

In `data_steward/init_env.sh` we set `UNIONED_DATASET_ID` not `UNIONED_EHR_DATASET_ID`.  This will probably fix that integration test error.